config/chmo.iris

```diff
@@ -17,3 +17,4 @@
 +(http://purl.obolibrary.org/obo/OBI_0000070):http://purl.obolibrary.org/obo/CHMO_0000678 Tip Enhanced Raman Scattering (nano-Raman)
 +(http://purl.obolibrary.org/obo/OBI_0000070):http://purl.obolibrary.org/obo/CHMO_0000373 TTransmission electron microscopy with electron energy loss spectroscopy
 +(http://purl.obolibrary.org/obo/OBI_0000070):http://purl.obolibrary.org/obo/CHMO_0000207 Wide-Angle X-ray Scattering
```

Comment:
Is it possible to add acronyms of the methods as well (perhaps as synonyms) - e.g. WAXS in this case.  Or do you already add these but perhaps not visible in git ?

Reply:
yes, but not with this approach. We would have to make an `chmo-ext.owl` file.

Reply:
> yes, but not with this approach. We would have to make an `chmo-ext.owl` file.

how this should be done - with another pull request?  e.g. WAXS is a synonym in CHMO_0000207 already